groundzero/src/components/About.jsx

```diff
@@ -1,9 +1,29 @@
 import React from 'react'
 import "../index.css"
+import styles from "../style";
+import { groundzero } from '../assets';
+import { aboutUs } from '../constants';
 
 const About = () => {
   return (
-    <div className="about" id="about">About</div>
+    <div
+    className={`${styles.paddingY} flex-col relative`}
+  >
+    <div className="grid place-items-center text-white xs:px-16 lg:px-40">
+      <div className="heading">
+        About Us
+        <br />
+        {aboutUs.map((aboutUs) => (
```

Comment:
Great use of map here, LGTM!
Perhaps we can use a linter such as Prettier to better format the code in the next iteration 😄